src/releasers/python.ts

```diff
@@ -76,6 +84,37 @@ export class Python extends ReleasePR {
       })
     );
 
+    const parsedPyProject = await this.getPyProject();
+    const pyProject = parsedPyProject?.project || parsedPyProject?.tool?.poetry;
+    if (pyProject) {
+      updates.push(
+        new PyProjectToml({
+          path: this.addPath('pyproject.toml'),
+          changelogEntry,
+          version: candidate.version,
+          packageName: packageName.name,
+        })
+      );
+
+      if (pyProject.name) {
```

Comment:
It seems like this code path is not covered by the test. Can you add the test case for it?